src/osdc/Objecter.cc

```diff
@@ -2838,10 +2838,9 @@ int Objecter::_calc_target(op_target_t *t, Connection *con, bool any_change)
 
   if (legacy_change || split_or_merge || force_resend) {
     t->pgid = pgid;
-    t->acting = acting;
```

Comment:
I would do `t->acting = std::move(acting);` here and replace `acting` with `t->acting` below.  A bit more churn but consistent with `up` and easier to spot the hand-off point.